index-pt-br.html

```diff
@@ -76,26 +74,62 @@
 
 <h2>Versão rápida</h2>
 
-<p>Nossa conferência é dedicada a fornecer uma experiência livre de assédio para todos, independentemente do sexo, idade, orientação sexual, deficiência, aparência física, tamanho, raça, ou religião. Nós não toleramos assédio dos participantes, sob qualquer forma. Linguagem e imagens sexuais não são adequadas em nenhum local da conferência, incluindo palestras, workshops, festas, Twitter e outros meios on-line. Os participantes da conferência que violarem estas regras podem ser convidados a se retirar ou ser expulsos <em>sem restituição</em> a critério dos organizadores da conferência.</p>
+<p>
+  Nossa conferência é dedicada a fornecer uma experiência de conferência livre de assédio para todos,
+  independentemente do sexo, identidade de gênero e expressão, idade, orientação sexual, deficiência,
+  aparência física, tamanho corporal, cor de pele, etnia ou religião (ou falta dela). Nós não toleramos o assédio
+  dos participantes da conferência, sob qualquer forma. Linguagem e imagens sexuais não são apropriadas para
+  qualquer local da conferência, incluindo palestras, workshops, festas, Twitter e outras mídias on-line.
+  Os participantes da conferência que violem estas regras pode ser punidos ou expulso da conferência
+  <em>sem restituição</em>, a critério dos organizadores da conferência.
+</p>
 
 <h2>Versão mais longa</h2>
+<p>
+  O assédio inclui comentários verbais ofensivos relacionados ao gênero, identidade de gênero e expressão,
+  idade, orientação sexual, deficiência, aparência física, tamanho corporal, cor de pele, etnia, religião, imagens
+  sexuais em espaços públicos, intimidação deliberada, perseguição, <em>stalking</em>, fotografias ou filmagens
+  constrangedoras, interrupção contínua das apresentações ou outros eventos, contato físico inadequado, e atenção
+  sexual indesejada.
+</p>
 
-<p>Será entendido como assédio: Comentários verbais ofensivos relacionados com sexo, idade, orientação sexual, deficiência, aparência física, tamanho, raça, religião, imagens sexuais em espaços públicos, intimidação deliberada, perseguição, assédio fotográfico, interrupção de palestra ou de alguma outra ação, contato físico inapropriado, e atenção sexual indesejada.</p>
-
-<p>Os participantes que forem advertidos por qualquer comportamento de assédio devem parar imediatamente.</p>
+<p>
+  Os participantes que sejam solicitados a interromperem qualquer comportamento de assédio devem cumprir
+  a solicitação imediatamente.
+</p>
 
-<p>Patrocinadores também estão sujeitos à política anti-assédio. Em particular, os patrocinadores não devem usar imagens, atividade ou outro material com contexto sexual. Recepcionistas de estandes (incluindo voluntários) não devem usar roupas/uniformes/trajes inadequadas (sexualmente atrativas), ou criar de alguma forma um ambiente sexualizado.</p>
+<p>
+  Os patrocinadores também estão sujeitos à políticas de anti-assédio. Em particular, os patrocinadores
```

Comment:
Typo: should be either "... sujeitos **às** políticas ..." or "... sujeitos **a** políticas ..." (without the grave accent).